ThaDeveloper/FastFoodFast_API
# 160651965 Database setup

#### What does this PR do?
Configure and setup database with postgresql
#### Description of Task to be completed?
Have the database setup script
Have a database manager.
#### How should this be manually tested?
1. `$ git clone https://github.com/ThaDeveloper/FastFoodFast_API`
2. Create databases:
- On your terminal run `$ psql postgres`
- `# create role role_name with createdb login password 'yourpassword';
- `# create database fast_food_fast_dev;`
- `# create database fast_food_fast_test;`
Ctrl + D to quit psql command line

3. `$ cd FastFoodFast_API` and  `$ pip3 install -r requirements.txt`

4. Create .env file with the following:
```
export FLASK_APP=run.py
export FLASK_ENV=development
export SECRET='yoursecret'
export USER=role_name
export PASSWORD=your_password
export HOST=localhost
export DEV_DATABASE=fast_food_fast_dev
export TEST_DATABASE=fast_food_fast_test
```
`$ source .env`
5. `$ python3 manage.py create_db`
Your database is now setup, can access psql terminal again to confirm

#### Any background context you want to provide?
Databse built in [psycopg2](https://pypi.org/project/psycopg2/)
